.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### PR Checklist
+- [ ] Have you updated the required documentation based on the current PR
+  changes
+- [ ] Have you written new tests for your core changes, as applicable?
+
+**Commit Formatting**
+
+Please follow the `seven rules of a great commit message`
+
+- Separate subject from body with a blank line
+- Limit the subject line to 50 characters
+- Capitalize the subject line
+- Do not end the subject line with a period
+- Use the imperative mood in the subject line
```

Comment:
I'm hoping the link to the _seven rules_ article explain _imperative mood_.  I certainly don't know what that means w/o looking it up.

Reply:
added link as well